app/src/main/java/org/mozilla/fenix/composables/HeadingText.kt

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.composables
+
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Default layout for the header of a screen section.
+ *
+ * @param text [String] to be styled as header and displayed.
+ * @param modifier [Modifier] to be applied to the [Text].
+ */
+@Composable
+fun HeadingText(
```

Comment:
I think we can just call this `Header`.

Reply:
Maybe `SectionHeader`?
What I tried in this patch also is to standardize some usecases.
We should have as few Composables as possible, each containing the specs for a particular usecase, reused all throughout the app.
A suggestive name and good description is a hard requirement.
So `SectionHeader` later followed by some other headers make sense to me.

Reply:
`SectionHeader` sounds good